docs/topics/i18n/translation.txt

```diff
@@ -1195,8 +1195,8 @@ a word or not::
     document.write(pluralidx(2));
     // true
 
-In the simplest case, if no custom pluralization is needed, this returns
-``false`` for the integer ``1`` and ``true`` for all other numbers.
```

Comment:
This *is* the simplest case, so I think revert it. Again not a judgment of the reader.